rfc/standalone.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // GetCacheKey returns the cache key for req.
 func GetCacheKey(req *http.Request) string {
-	return fmt.Sprintf("%s-%s-%s", req.Method, req.Host, req.URL.Path)
+	return fmt.Sprintf("%s-%s-%s", req.Method, req.Host, req.URL.RequestURI())
```

Comment:
Should we have to remove empty querystring filters to avoid `/path?q1=&q2=&q3=&qX=` different than `/path` ? 
And return the same value for `/path?q1=value&q2=value2&q3=` and `/path?q1=value&q2=value2` for example

What do you think about it ?

Reply:
I don't think it's a good idea. Given that HTTP query parameters are untyped, there are implementations using keys without values in order to represent boolean values. Thus, I could imagine how `/path?q1=value&q2=value2&q3=` could serve different content than `/path?q1=value&q2=value2` if you substitute `q3` with `enable_beta`, `enable_preview`, `testing`, etc.

Reply:
You got the point